frontend/src/components/worksheets/items/TableItem/BundleRow.js

```diff
@@ -317,16 +310,7 @@ class BundleRow extends Component {
                                 showDetail={showDetail}
                                 handleDetailClick={this.handleDetailClick}
                                 editPermission={editPermission}
-                                onOpen={() => {
-                                    const node = document.getElementById(
-                                        'codalab-worksheet-item-' +
-                                            this.state.focusIndex +
-                                            'subitem-' +
-                                            this.props.rowIndex +
-                                            '-detail',
-                                    );
-                                    node.scrollIntoView({ block: 'center' });
-                                }}
+                                onOpen={() => {}}
```

Comment:
Why remove this?

Reply:
it was created by me for the bundle detail scrolling stuff. Since I switched a method now, I removed the old code